lib/index.js

```diff
@@ -3,23 +3,39 @@
  */
 'use strict';
 
-const {documentLoader} = require('bedrock-jsonld-document-loader');
-const jsigs = require('jsonld-signatures');
-const {Ed25519Signature2018} = jsigs.suites;
+const {jsonLdDocumentLoader} = require('bedrock-jsonld-document-loader');
+const {Ed25519Signature2020} = require('@digitalbazaar/ed25519-signature-2020');
 const {validateOperation, runOperation} = require('webkms-switch');
 const keystores = require('./keystores.js');
 const BedrockKmsModuleManager = require('./BedrockKmsModuleManager.js');
 const MongoDBKeyDescriptionStorage =
   require('./MongoDBKeyDescriptionStorage.js');
+
+const ed25519sig2018 = require('ed25519-signature-2018-context');
+const ed25519sig2020 = require('ed25519-signature-2020-context');
+const x25519 = require('x25519-key-agreement-2020-context');
+
 require('bedrock-did-context');
 require('bedrock-veres-one-context');
 
-const didIo = require('did-io');
+jsonLdDocumentLoader.addStatic(ed25519sig2018.constants.CONTEXT_URL,
+  ed25519sig2018.contexts.get(ed25519sig2018.constants.CONTEXT_URL));
+
+jsonLdDocumentLoader.addStatic(ed25519sig2020.constants.CONTEXT_URL,
+  ed25519sig2020.contexts.get(ed25519sig2020.constants.CONTEXT_URL));
+
+jsonLdDocumentLoader.addStatic(x25519.constants.CONTEXT_URL,
+  x25519.contexts.get(x25519.constants.CONTEXT_URL));
```

Comment:
I have to say that this feels like it's going to get duplicated in a number of places. Like a bedrock-did-io or something would make sense here. Also,, is this is a problem to add these at the top-level of the module? Should they be added in a bedrock event instead? I'm not familiar/don't remember the pattern here, just noting that there may be an issue here.

Reply:
First question I have is: Is support for the 2018 suite still being maintained?  I thought we were just dropping it everywhere and making breaking releases.

Second, `bedrock-security-context@4` addresses this: https://github.com/digitalbazaar/bedrock-security-context/blob/main/CHANGELOG.md

Reply:
@JSAssassin I think you want to add a peerDep for bedrock-security-context@4 and require it as with some of the other context modules.

Also, I'll point out that the bedrock-did-context peer dep here is now out of date: https://github.com/digitalbazaar/bedrock-did-context/blob/main/CHANGELOG.md

Reply:
Updated here c935241